### PR TITLE
ICMSLST-1897 Add Home Officer Case Officer group

### PR DESCRIPTION
**Pull request by MattHolmes123**
Changes:
  - Add "Home Office Case Officer" Group record.
  - Add new importer_regulator permission.
  - Add ImporterListHomeOfficeView.
  - Add ImporterDetailHomeOfficeView.
  - Update all section5 views to work with Home Office Case Officer.
  - Add ho_admin_user test system user
  - Add ho_admin_user unittest fixture.
  - Refactor several templates to work with new group.

### CODE REVIEW

File: web/permissions/perms.py [hunk 1]
```diff
@@ -28,6 +28,8 @@ class SysPerms(PermissionTextChoice):
     importer_access = "web.importer_access", "Can act as an importer"
     exporter_access = "web.exporter_access", "Can act as an exporter"
     ilb_admin = "web.ilb_admin", "Is an ILB administrator"
+    importer_regulator = "web.importer_regulator", "Is an Importer Regulator"
```
[the remark this answers is not in the document]
Comment (MattHolmes123) [reply web/permissions/perms.py:31]: If you think of a better name let me know.

File: web/tests/domains/importer/test_views.py [hunk 2]
```diff
@@ -97,6 +97,70 @@ def test_get(self):
         assert importer == self.importer
 
 
+class TestImporterListHomeOfficeView(AuthTestCase):
```
Comment (marcuspp) [inline web/tests/domains/importer/test_views.py:100]: Could we turn on `STRICT_TEMPLATES` for these tests? Its just some of the fixes in the jinja2 strict task showed up issues with `partial/agents.html` not having everything it needed.

**Review by marcuspp — APPROVED** [review]

File: web/tests/domains/importer/test_views.py (hunk 2 above)
Comment (MattHolmes123) [reply web/tests/domains/importer/test_views.py:100]: Fixed the DetailView not the list view.

File: web/templates/partial/importer/detail.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      {{ importer.main_importer.display_name }}</a>.
+  </div>
+{% endif %}
+
+<h3>{{ importer.get_type_display() }}</h3>
+
+{% if importer.is_organisation() %}
+  <dl>
+    <dt class="bold">Organisation Name</dt>
+    <dd>{{ importer.name }}
```
[the remark this answers is not in the document]
Comment (MattHolmes123) [reply web/templates/partial/importer/detail.html]: sorted ta

**Pull request merged**